index.html

```diff
@@ -358,68 +470,84 @@ <h2>
     </section>
     <section data-dfn-for="Navigator">
       <h2>
-        Navigator Interface extension
+        Extensions to the {{Navigator}} interface
       </h2>
-      <p>
-        This partial interface defines an extension to the Navigator interface.
-      </p>
       <pre class="idl">
         [Exposed=Window]
         partial interface Navigator {
           sequence&lt;Gamepad?&gt; getGamepads();
         };
       </pre>
-      <dl>
-        <dt>
-          <dfn>getGamepads()</dfn>
-        </dt>
-        <dd>
-          <p class="note">
-            The gamepad state returned from {{Navigator/getGamepads()}} does
-            not reflect disconnection or connection until after the
-            <a>gamepaddisconnected</a> or <a>gamepadconnected</a> events have
-            fired.
-          </p>
-          <p>
-            If the <a>current settings object</a>'s [=environment settings
-            object / responsible document=] is not <a>allowed to use</a> the
-            "`gamepad`" permission, then [=exception/throw=] a
-            {{"SecurityError"}} {{DOMException}}.
-          </p>
-        </dd>
-        <dd>
-          Retrieve a snapshot of the data for the the currently connected and
-          interacted-with gamepads. Gamepads MUST only appear in the list if
-          they are currently connected to the <a>user agent</a>, and at least
-          one device has been interacted with by the user. If no devices have
-          been interacted with, devices MUST NOT appear in the list to avoid a
-          malicious page from fingerprinting the user. The length of the array
-          returned MUST be one more than the maximum index value of the Gamepad
-          objects returned in the array. The entries in the array MUST be the
-          set of Gamepad objects that are visible to the current page, with
-          each Gamepad present at the index in the array specified by its
-          {{Gamepad/index}} attribute. Array indices for which there is no
-          connected Gamepad with the corresponding index should return null.
-          <p>
-            The gamepad state returned from getGamepads() does not reflect
-            disconnection or connection until after the
-            <a>gamepaddisconnected</a> or <a>gamepadconnected</a> events have
-            fired.
-          </p>
-          <p>
-            As an example, if there is one connected gamepad with an index of
-            1, then the following code snippet describes the expected behavior:
-          </p>
-          <pre class="example highlight">
+      <p>
+        Instances of {{Navigator}} are created with the internal slots described in the following table:
+      </p>
+      <table class="simple" dfn-for=Navigator dfn-type=attribute>
+        <tr>
+          <th>Internal slot
+          <th>Initial value
+          <th>Description (non-normative)
+        <tr>
+          <td><dfn>[[\hasGamepadGesture]]</dfn>
```

Comment:
These should be: 

```suggestion
          <td><dfn data-dfn-for="Navigator">[[\hasGamepadGesture]]</dfn>
```

And then... to link... a little ugly... I know: 

```
{{Navigator/[[hasGamepadGesture]]}}
```